include/TPP/Passes.h

```diff
@@ -22,6 +22,24 @@ class FuncDialect;
 } // namespace func
 } // namespace mlir
 
+namespace mlir {
```

Comment:
nit: this is getting silly. We should only have one `namespace mlir` and all others inside. :)